dartsim/src/ShapeFeatures.cc

```diff
@@ -252,7 +252,7 @@ Vector3d ShapeFeatures::GetEllipsoidShapeRadii(
 Identity ShapeFeatures::AttachEllipsoidShape(
     const Identity &_linkID,
     const std::string &_name,
-    const Vector3d _radii,
+    const Vector3d &_radii,
```

Comment:
I think this is a breaking change, so we wouldn't want to do it in this branch. Instead, it should go into `main`. Also, to do this, we'd have to change the signatures in the `AttachEllipsoidShapeFeature` (https://github.com/gazebosim/gz-physics/blob/8ee8028861567303cdba2602c9ba79c34a5ab542/include/gz/physics/EllipsoidShape.hh)

Reply:
I've applied these API changes in https://github.com/gazebosim/gz-physics/pull/521, targeting `main`

Reply:
I reverted this change in https://github.com/gazebosim/gz-physics/pull/515/commits/39e475315769ed20a77e5c2e8ff1e3fb32795f30